src/gridstack-dd.ts

```diff
@@ -132,7 +132,7 @@ GridStack.prototype._setupAcceptWidget = function(this: GridStack): GridStack {
         if (node?.grid === this) return true;
         if (!this.opts.acceptWidgets) return false;
         // prevent deeper nesting until rest of 992 can be fixed

```

Comment:
I guess my comment can be removed now that 992 is fixed

Reply:
I delete unused comment.